### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by shuplenkov**
## What was done?

DPP updated to 0.20.0-dev.1
Wallet-lib updated to 7.20.0-dev.2

## How Has This Been Tested?

Tests

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/SDK/Client/Platform/Platform.ts [hunk 1]
```diff
@@ -132,12 +133,17 @@ export class Platform {
...
+        this.dppForParsingContracts = new DashPlatformProtocol();
+        const stateRepository = new StateRepository(this.client, this.dppForParsingContracts);
 
         this.dpp = new DashPlatformProtocol({
             stateRepository,
             ...options,
         });
     }
+
+    async initialize() {
```
Comment (shumkov) [inline src/SDK/Client/Platform/Platform.ts:145]: Let's call initialize in Client methods - they are all async so we can init DPP in lazy way.

File: src/SDK/Client/Platform/methods/contracts/broadcast.ts [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { signStateTransition } from "../../signStateTransition";
  * @return dataContract
  */
 export default async function broadcast(this: Platform, dataContract: any, identity: any): Promise<any> {
+    await this.initialize();
```
Comment (shumkov) [inline src/SDK/Client/Platform/methods/contracts/broadcast.ts:14]: Let's remove `Client#initialize` method and call `this.dpp.initialize`?

Comment (shuplenkov) [reply src/SDK/Client/Platform/methods/contracts/broadcast.ts:14]: You mean platform#initialize?

**Review by shumkov — APPROVED** [review]
LGTM! 👍

**Pull request merged**